<properties
 pageTitle="RESOURCE_DOES_NOT_EXIST"
 description="RESOURCE_DOES_NOT_EXIST"
 service="Microsoft.Databricks"
 resource="Microsoft.Databricks/workspaces"
 authors="AzureData_AzureDatabricks"
 ms.author="AzureData_AzureDatabricks"
 displayOrder=""
 selfHelpType="TSG_Content"
 supportTopicIds=""
 resourceTags=""
 productPesIds=""
 cloudEnvironments="public, fairfax, usnat, ussec"
 articleId="f4e5c059-2e77-4ff2-afdc-282d1cbb9a05"
 ownershipId="AzureData_AzureDatabricks"
/>

# RESOURCE_DOES_NOT_EXIST

### Symptom

Adding user in the group API results with error `RESOURCE_DOES_NOT_EXIST`.

### Troubleshooting & Solution

When we use group API to add any user in the group list, but the user or the group does not exist, then will get this error.
 
- Check if the user exists in Azure AD group or in the Databricks user list.
- Check if the group exists
- Check if the group name is correct 
- Check if the user id is correct 

Since error indicates that the value you are passing is not available or not correct, you have to recheck the value and rerun the API again. 	

### Next Steps

- If issue persists, ask SME on Ava and create Databricks SF ticket if needed."
